app/controllers/AiModelController.scala

```diff
@@ -0,0 +1,224 @@
+    {
+      for {
+        _ <- userService.assertIsSuperUser(request.identity)
+        aiModelIdValidated <- ObjectId.fromString(aiModelId)
+        aiModel <- aiModelDAO.findOne(aiModelIdValidated) ?~> "aiModel.notFound" ~> NOT_FOUND
+        jsResult <- aiModelService.publicWrites(aiModel)
+      } yield Ok(jsResult)
+    }
+  }
+
+  def readAiInferenceInfo(aiInferenceId: String): Action[AnyContent] = sil.SecuredAction.async { implicit request =>
+    {
+      for {
+        _ <- userService.assertIsSuperUser(request.identity)
+        aiInferenceIdValidated <- ObjectId.fromString(aiInferenceId)
+        aiInference <- aiInferenceDAO.findOne(aiInferenceIdValidated) ?~> "aiInference.notFound" ~> NOT_FOUND
+        jsResult <- aiInferenceService.publicWrites(aiInference, request.identity)
+      } yield Ok(jsResult)
+    }
+  }
+
+  def listAiModels: Action[AnyContent] = sil.SecuredAction.async { implicit request =>
+    {
+      for {
+        _ <- userService.assertIsSuperUser(request.identity)
+        aiModels <- aiModelDAO.findAll
+        jsResults <- Fox.serialCombined(aiModels)(aiModelService.publicWrites)
+      } yield Ok(Json.toJson(jsResults))
+    }
+  }
+
+  def listAiInferences: Action[AnyContent] = sil.SecuredAction.async { implicit request =>
+    {
+      for {
+        _ <- userService.assertIsSuperUser(request.identity)
+        aiInferences <- aiInferenceDAO.findAll
+        jsResults <- Fox.serialCombined(aiInferences)(inference =>
+          aiInferenceService.publicWrites(inference, request.identity))
+      } yield Ok(Json.toJson(jsResults))
+    }
+  }
+
+  def runTraining: Action[RunTrainingParameters] = sil.SecuredAction.async(validateJson[RunTrainingParameters]) {
+    implicit request =>
+      for {
+        _ <- userService.assertIsSuperUser(request.identity)
+        trainingAnnotations = request.body.trainingAnnotations
+        _ <- bool2Fox(trainingAnnotations.nonEmpty || request.body.workflowYaml.isDefined) ?~> "aiModel.training.zeroAnnotations"
+        firstAnnotationId <- trainingAnnotations.headOption.map(_.annotationId).toFox
+        annotation <- annotationDAO.findOne(firstAnnotationId)
+        dataset <- datasetDAO.findOne(annotation._dataset)
+        dataStore <- dataStoreDAO.findOneByName(dataset._dataStore) ?~> "dataStore.notFound"
+        _ <- Fox
+          .serialCombined(request.body.trainingAnnotations.map(_.annotationId))(annotationDAO.findOne) ?~> "annotation.notFound"
+        modelId = ObjectId.generate
+        organization <- organizationDAO.findOne(request.identity._organization)
+        jobCommand = JobCommand.train_model
+        commandArgs = Json.obj(
+          "training_annotations" -> Json.toJson(trainingAnnotations),
+          "organization_name" -> organization.name,
+          "model_id" -> modelId,
+          "workflow_yaml" -> request.body.workflowYaml
+        )
```

Comment:
Shouldn't this include a `"webknossos_token" & "user_auth_token"`? For me, the run fails because the annotation could not be downloaded by the worker due to a missing token. Here are the command args send to there worker:

```
Starting train_model job run 665586e97d0a009e0e7e59a2__train_model__1716881129997 with params ['model_id: 665586e97d0a00920e7e59a1', 'workflow_yaml: None', 'organization_name: sample_organization', "training_annotations: [{'mag': [1, 1, 1], 'annotationId': '665586357d0a003f0e7e599c', 'colorLayerName': 'color', 'segmentationLayerName': 'segmentation'}]", 'user_auth_token: ***'].
```

Reply:
The user_auth_token is added automatically to every job’s args in `JobService.parameterWrites`. It should also be enough, no jobs should need webknossos_token anymore. Did you pull the latest changes on the worker side?